aggregate_classroom_info.py

```diff
@@ -0,0 +1,44 @@
+import json
+import os
+from gAPI import Create_Service
+from course_util import (CourseFetcher, write_submissions)
+from concurrent.futures import (
+    as_completed,
+    ThreadPoolExecutor
+)
+
+TOKEN_DIR = "data/tokens"
+CLIENT_SECRET_FILE = 'client_secret.json'
+API_SERVICE_NAME = 'classroom'
+API_VERSION = 'v1'
+SCOPES = ['https://www.googleapis.com/auth/classroom.coursework.students https://www.googleapis.com/auth/classroom.courses https://www.googleapis.com/auth/classroom.rosters https://www.googleapis.com/auth/classroom.profile.emails https://www.googleapis.com/auth/classroom.rosters']
+
+def get_classroom_data(token):
+    print("\n-------------- Fetching classroom data with %s ----------------" % token)
+
+    # Fetch all submissions that are accessible with the specified token
+    classroom = Create_Service(CLIENT_SECRET_FILE, API_SERVICE_NAME, API_VERSION, SCOPES, token_path=os.path.join(TOKEN_DIR, token))
+    fetcher = CourseFetcher(classroom=classroom)
+    submissions = fetcher.fetch()
```

Comment:
Once `course_util.py` is refactored to use a class, the actual aggregation part is pretty simple.